mvc-framework/engine/mandarineMvcFrameworkStarter.ts

```diff
@@ -244,22 +244,24 @@ export class MandarineMvcFrameworkStarter {
       }
     };
 
+
+    router = router.options(route, <any>responseHandler);
     switch (routingAction.actionType) {
       case Mandarine.MandarineMVC.HttpMethods.GET:
         return router.get(route, <any>responseHandler);
       case Mandarine.MandarineMVC.HttpMethods.POST:
         return router.post(route, <any>responseHandler);
       case Mandarine.MandarineMVC.HttpMethods.DELETE:
         return router.delete(route, <any>responseHandler);
-      case Mandarine.MandarineMVC.HttpMethods.OPTIONS:
-        return router.options(route, <any>responseHandler);
       case Mandarine.MandarineMVC.HttpMethods.PUT:
         return router.put(route, <any>responseHandler);
       case Mandarine.MandarineMVC.HttpMethods.PATCH:
         return router.patch(route, <any>responseHandler);
       case Mandarine.MandarineMVC.HttpMethods.HEAD:
         return router.head(route, <any>responseHandler);
     }
+
+    return router;
```

Comment:
Why do we need to return the router here? What's the reasoning behind "it was working before without returning it"?

Reply:
after removing the OPTIONS case, the case statement list is not exhaustive anymore. 

moving the `return router; ` to the default case may be a good idea for readability :thinking:

Reply:
sounds good to me then